vars/updateBuildFailureIssues.groovy

```diff
@@ -70,7 +73,7 @@ void call(Map args = [:]) {
                 closeComment: ghIssueBody,
                 label: "autocut,v${currentVersion}"
             )
+            sleep(time:3, unit:'SECONDS')
```

Comment:
Why move sleep?

Reply:
See the PR description. Sleep is needed only when gh cli is called to avoid back to back api hits. Right now it just sleeps for no reason https://build.ci.opensearch.org/view/Test/job/integ-test-notification/13/console